src/waveform/renderers/waveformrenderersignalbase.cpp

```diff
@@ -188,19 +197,16 @@ void WaveformRendererSignalBase::getGains(float* pAllGain,
         // Only adjust low/mid/high gains if EQs are enabled.
         if (m_pEQEnabled->get() > 0.0) {
             if (m_pLowFilterControlObject &&
-                m_pMidFilterControlObject &&
-                m_pHighFilterControlObject) {
+                    m_pMidFilterControlObject &&
+                    m_pHighFilterControlObject) {
                 lowGain = static_cast<CSAMPLE_GAIN>(m_pLowFilterControlObject->get());
                 midGain = static_cast<CSAMPLE_GAIN>(m_pMidFilterControlObject->get());
                 highGain = static_cast<CSAMPLE_GAIN>(m_pHighFilterControlObject->get());
             }
 
-            lowGain *= static_cast<CSAMPLE_GAIN>(
-                    factory->getVisualGain(WaveformWidgetFactory::Low));
-            midGain *= static_cast<CSAMPLE_GAIN>(
-                    factory->getVisualGain(WaveformWidgetFactory::Mid));
-            highGain *= static_cast<CSAMPLE_GAIN>(
-                    factory->getVisualGain(WaveformWidgetFactory::High));
+            lowGain *= m_lowVisualGain;
+            midGain *= m_midVisualGain;
+            highGain *= m_highVisualGain;
```

Comment:
```suggestion
            lowVisualGain *= m_lowVisualGain;
            midVisualGain *= m_midVisualGain;
            highVisualGain *= m_highVisualGain;
```
Left hand side is also a visual gain